pubspec.yaml

```diff
@@ -44,7 +44,7 @@ dependencies:
   streaming_shared_preferences: ^2.0.0
   # connectivity
   connectivity: ^3.0.6
-#  connectivity_plus: ^2.3.4
+  #  connectivity_plus: ^2.3.4
```

Comment:
remove this since we are not really using it

Reply:
Done